en/sql/tuning.rst

```diff
@@ -368,6 +386,228 @@ If you ensure that left table's row number is a lot smaller than the right table
+
+.. code-block:: sql
+
+    drop table if exists t2;
+    create table t2 (col1 int, col2 int, col3 int, col4 int);
+    insert into t2 select mod(rownum,20), mod(rownum,80), rownum, rownum from dual connect by level <= 4000;
+    create index idx on t2(col1,col2,col3);
+    update statistics on t2;
+     
+    ;plan detail
+    select /*+ recompile */ count(*) from t2 where col1 = 1 and col3 = 1 and col4 = 1;
+
+::
+
+    Join graph terms:
+    term[0]: [dba.t2].col4=1 (sel 0.00025)
+    term[1]: [dba.t2].col3=1 (sel 0.0125)
+    term[2]: [dba.t2].col1=1 (sel 0.05)
+
+    Query plan:
+     
+    iscan
+        class: t2 node[0]
+        index: idx term[2]
+        filtr: term[1]
+        sargs: term[0]
+        cost:  4 card 1
+
+You can check the predicates in the execution plan through the information in **Join graph terms**. **index: idx term[2]** is the predicate for searching the non-leaf node during the index vertical scan. The **filtr: term[1]** predicate is for filtering the key in the leaf node during the index horizontal scan. When accessing the heap and filtering data, the **sargs: term[0]** predicate is used to filter.
+
+::
+
+    ;info stats t2
+
+::
+
+     Attribute: col1 (integer)
+        Number of Distinct Values: 20
+        B+tree statistics:
+            BTID: { 0 , 5760 }
+            Cardinality: 4000 (20,4000,4000) , Total pages: 11 , Leaf pages: 9 , Height: 2
+
+The cost of an index scan is calculated as follows:
+
+    #. Cost of reading a page = predicted non-leaf node pages accessed + leaf node pages + heap pages
+    #. CPU cost of a repeated routine = predicted number of leaf node keys accessed
+
+The number of pages in a non-leaf node is 1 because the index **Height** of the statistics is 2 and the number of pages to be read excluding the leaf node is **2 - 1**. The number of leaf node pages is obtained by multiplying **Leaf pages** by the selectivity of the predicate **index: idx term[2]** used to search the non-leaf node. Since it is **MAX(9 X 0.05, 1)**, the number of leaf node pages to be read is 1. The number of heap pages to be read is obtained by multiplying the total number of pages in the table by the selectivity of **index** and **filtr** conditions. Here, it is calculated as **MAX(9 X 0.05 X 0.0125,1)**, so it is calculated that 1 page must be read. Finally, the CPU cost is obtained by multiplying the selectivity of **index: idx term[2]** by the CPU weight from the total number of rows in the table. **If you calculate it as **MAX(4000 X 0.05 X 0.0025,1)**, you get 1, and if you add up all the results so far, you can see that the cost is 4. If you look at **cost: 4 card 1** in the execution plan, you can see that the cost is 4 and the expected number of rows is 1.
```

Comment:
```suggestion
The number of pages in a non-leaf node is 1 because the index **Height** of the statistics is 2 and the number of pages to be read excluding the leaf node is **2 - 1**. The number of leaf node pages is obtained by multiplying **Leaf pages** by the selectivity of the predicate **index: idx term[2]** used to search the non-leaf node. Since it is **MAX(9 X 0.05, 1)**, the number of leaf node pages to be read is 1. The number of heap pages to be read is obtained by multiplying the total number of pages in the table by the selectivity of **index** and **filtr** conditions. Here, it is calculated as **MAX(9 X 0.05 X 0.0125, 1)**, so it is calculated that 1 page must be read. Finally, the CPU cost is obtained by multiplying the selectivity of **index: idx term[2]** by the CPU weight from the total number of rows in the table. If you calculate it as **MAX(4000 X 0.05 X 0.0025, 1)**, you get 1, and if you add up all the results so far, you can see that the cost is 4. If you look at **cost: 4 card 1** in the execution plan, you can see that the cost is 4 and the expected number of rows is 1.
```